.github/workflows/build.yml

```
@@ -5,13 +5,13 @@ jobs:
     runs-on: macos-10.15
     steps:
     - uses: actions/checkout@v2
-    - id: configure-cmake
+    - name: Configure cmake`
```

Is that ` intentional?
```suggestion
    - name: Configure cmake
```

---

no, type , I am on the notebook, the keyboard is ... (I hate the MBP16, the 2015 was so much better)